recipes/install-redis-server.rb

```diff
@@ -0,0 +1,6 @@
+# Cookbook Name:: mh-opsworks-recipes
+# Recipe:: install-redis-server
+
+::Chef::Recipe.send(:include, MhOpsworksRecipes::RecipeHelpers)
+install_package('redis-server')
```

Comment:
ok, i'm expiring keys every 24h. if i ended up throwing more info in redis i'll review persistence and expiration, but for now it should not grow too much. it mainly has a fixed set of capture agents (that does not grow during a semester) and user sessions - and this app is for internal use, so the number of users should be limited.